test/plugin-track.test.js

```diff
@@ -105,7 +104,6 @@ describe('Plugin track tests', () => {
       'pixiTextToSvg',
       'q',
       'rangeQuery2d',
-      'reduce',
```

Comment:
This is my main concern. We export every since utility function in higlass for plugins (i.e., as a part of HiGlass's public API), so this would technically be a breaking change. 

I have no idea how much each of these functions is used in practice. I'm certain that there are some which are commonly used for higlass-specific stuff (e.g., chromsizes stuff).

Reply:
A more thoughtful way of doing this would be in three parts:

1.) section off utilities which are abandoned internally (e.g., move all to a `abandoned-utils-for-plugins.js` and re-export)
2.) scrutinize our "public API" to decide what makes the cut for a breaking release
3.) make a minor release with well documented removal of various utilities

I'm not suggesting these changes for no reason. It would make it easier to achieve #1162 if we could safely "ignore" needing to worry about various utilities. Second, it would tame the API surface area of the project.

Reply:
More thoughtful way seems reasonable. Not sure I get the minor release removal. Wouldn't we just drop everything unused in v2?

Another alternative is to put these utilities into a separate `higlass-utils` package.

At first glance some of these utils are for testing purposes so they could go into a `higlass-testing` package or something to that effect?